frontend/src/components/map/Controls.tsx

```diff
@@ -13,23 +13,27 @@ export default function Controls(props:ControlsProps) {
     const [lng, setLng] = useState<number>(0)
     const [lat, setLat] = useState<number>(0)
 
-    function onMove() {
-        if (spotmap) {
-            setLat(spotmap?.getCenter().lat)
-            setLng(spotmap?.getCenter().lng)
-
-            props.handleCenterCoordinatesChange({
-                lat: parseFloat(spotmap?.getCenter().lat.toFixed(4)),
-                lng: parseFloat(spotmap?.getCenter().lng.toFixed(4))
-            })
-        }
-    }
 
     useEffect(() => {
         if (!spotmap) return;
+
+        function onMove() {
+            if (spotmap) {
+                setLat(spotmap?.getCenter().lat)
+                setLng(spotmap?.getCenter().lng)
+
+                props.handleCenterCoordinatesChange({
+                    lat: parseFloat(spotmap?.getCenter().lat.toFixed(4)),
+                    lng: parseFloat(spotmap?.getCenter().lng.toFixed(4))
+                })
+            }
+        }
+
         spotmap.on("move", onMove)
         onMove()
-    })
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Aus den Docs
> ✅ Call Hooks from React function components.
    ✅ Call Hooks from custom Hooks (we’ll learn about them [on the next page](https://reactjs.org/docs/hooks-custom.html)).
    
Scheint das keins davon verletzt wird? Ich glaube für genau solche Fällen wo ein anderer Dev sich fragt "but why", wäre es gut des in nem kurzen Kommentar zu dokumentieren